src/tests/ftest/datamover/posix_subsets.py

```diff
@@ -52,6 +53,13 @@ def run_dm_posix_subsets(self, tool):
         # Create 1 pool
         pool1 = self.create_pool()
 
+	# create dfuse containers to test copying to dfuse subdirectories
+        dfuse_cont1 = self.create_cont(pool1)
+        dfuse_cont2 = self.create_cont(pool1)
+        dfuse_cont1_dir = str(join(self.dfuse.mount_dir.value, pool1.uuid, dfuse_cont1.uuid))
```

Comment:
I think `join` always returns a `str`, right?
```suggestion
        dfuse_cont1_dir = join(self.dfuse.mount_dir.value, pool1.uuid, dfuse_cont1.uuid)
```